### PR TITLE
Add GEO_DISTANCE_WEIGHT to configuration file

### PR DESCRIPTION
**Pull request by YoannRonat**
Provides a more accessible tuning of the importance of geo_distance in
final scores

### CODE REVIEW

**Comment by jdesboeufs** [comment]
Maybe you can add some words about this in the documentation?
Thank you for your PR ❤️

**Comment by YoannRonat** [comment]
In the same PR ?

**Comment by yohanboniface** [comment]
@YoannRonat yes, just add an entry in the "Advanced Settings" [here](https://github.com/addok/addok/blob/master/docs/config.md).
And a unittest would be great too (ping me if you need some hints) :)

File: docs/config.md [hunk 1]
```diff
@@ -259,6 +259,11 @@ The max inherent score of a document in the final score.
 
     IMPORTANCE_WEIGHT = 0.1
 
+#### GEO_DISTANCE_WEIGHT (float)
+The max inherent score of geographical distance to the provided center (if any) in the final score.
+
+    GEO_DISTANCE_WEIGHT = 2.0
```
Comment (yohanboniface) [inline docs/config.md]: We usually put here the default value :)

Comment (YoannRonat) [reply docs/config.md]: Fixed :wink:

**Comment by yohanboniface** [comment]
Thanks!

(fixes #486 )

**Pull request merged**